src/functions-inl.h

```diff
@@ -0,0 +1,57 @@
+#pragma once
```

Comment:
Could not get why this complete new header file is added but not used anywhere.

Reply:
Good catch. That was refactored out, but came back when I rebased. Will delete it